src/vm/database/marf.rs

```diff
@@ -62,10 +62,7 @@ pub fn in_memory_marf() -> MarfedKV<SqliteConnection> {
     MarfedKV { chain_tip, marf, side_store }
 }
 
-/// If chain_tip is None, this will try to figure out a reasonable value for a starting
-///   chain_tip. If the MARF already has some chain_tips, then it selects ordinal 0.
-///   Otherwise, it uses the block_sentinel.
-pub fn sqlite_marf(path_str: &str, chain_tip: Option<BlockHeaderHash>) -> Result<MarfedKV<SqliteConnection>> {
+pub fn sqlite_marf(path_str: &str, miner_tip: Option<&BlockHeaderHash>) -> Result<MarfedKV<SqliteConnection>> {
```

Comment:
It's usually better for the function argument to accept an owned reference in situations like this -- that allows the caller to decided whether or not a clone() is necessary -- otherwise, the function will _always_ need to clone, even if the caller has an owned reference that it's not ever using again